common-graphql/src/main/scala/queries/common/ManualSequenceGQL.scala

```diff
@@ -0,0 +1,269 @@
+// Copyright (c) 2016-2022 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package queries.common
+
+import clue.GraphQLOperation
+import clue.annotation.GraphQL
+import io.circe.Decoder
+import lucuma.core.math.Angle
+import lucuma.core.math.Offset
+import lucuma.core.model.sequence.ManualConfig
+import lucuma.schemas.ObservationDB
+
+// gql: import lucuma.schemas.decoders._
+// gql: import lucuma.ui.reusability._
+
+object ManualSequenceGQL {
+
+  @GraphQL
+  trait SequenceSteps extends GraphQLOperation[ObservationDB] {
+    val document = """
+      query($programId: ProgramId!) {
```

Comment:
general question. this queries are validated by the scalafix steps right? Just asking to confirm these are validated at compile time

Reply:
Right, they are checked against the schema by the scalafix rule, which is run whenever we compile.